spec/dummy_winrm_service.rb

```diff
@@ -0,0 +1,24 @@
+module Dummy
```

Comment:
yeah because the `HTTPClient` is a couple classes deep, accessing via a double got awkward and this makes it easier to share in different tests.